src/gl/shader_sources.js

```diff
@@ -1,22 +1,6 @@
 // Generated from GLSL files, don't edit!
```

Comment:
Weird this file changed, I think it just re-compiled in a different order this time? There aren't/shouldn't be any changes as far as I know.